Media/FFmpegMovie/FFmpegMovieFrame.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace OpenOrchestra\Media\FFmpegMovie;
+
+use FFMpeg\FFMpeg;
+use FFMpeg\Coordinate\TimeCode;
+
+/**
+ * Class FFmpegMovieFrame
+ */
+class FFmpegMovieFrame implements FFmpegMovieFrameInterface
+{
+    protected $ffmpeg;
+
+    /**
+     * FFmpegMovieFrame constructor.
+     */
+    public function __construct()
+    {
+        $this->ffmpeg = FFMpeg::create();
```

Comment:
instead of creating the ffmpeg instance in the constructor, wouldn't it be possible to use a factory to create this service and inject it inside this class ?